scss/components/_common.scss

```diff
@@ -97,60 +98,144 @@ body {
 .slide {
   margin-bottom: 100px;
   &__description {
-    font-size: 50px;
-    font-family: 'Raleway',sans-serif;
-    font-weight: 300;
+    @include font('Raleway', 300, 50px);
     margin: 10px 0;
-    color: $color-txt-primary;
-    font-size: 50px;    
-    padding: 6px 230px 13px 0;
+    color: $color-txt-primary;  
+    padding: 6px 254px 14px 0;
     line-height: 65px;
+    word-spacing: 4.1px;
+  }
+  &__title {
+    position: relative;
+
+    &::before {
+      @include pseudo ("", 210px, 10px);
+      background: url(images/title-line.png) no-repeat;
+      left: 0;
+      bottom: 5px;
+    }
   }
 }
 
 .btn-wrapper {
-  padding: 35px 0;
-
-    .btn {
-      width: 151px;
-      text-align: center;
-      font-size: 17px;
-      font-family: 'Raleway',sans-serif;
-      font-weight: 600;
-      color: $color-txt-primary; 
-      user-select: none;
-      text-transform: uppercase;
-      background: transparent; 
-      padding: 14px 0;
-      outline: none;
-      margin-right: 17px;
-      border: 1px solid $color-txt-primary;
-
-      &:hover {
-        background-color: $color-btn-hover;
-        border-color: $color-btn-hover;
-      }
+  padding: 34px 0;
+
+  .btn {
+    width: 151px;
+    text-align: center;
+    @include font('Raleway', 600, 17px);
+    color: $color-txt-primary; 
+    user-select: none;
+    text-transform: uppercase;
+    background: transparent; 
+    padding: 14px 0;
+    outline: none;
+    margin-right: 17px;
```

Comment:
Use some kind of modificator for indents, in perspective not all your buttons will have an indent.